lang/ada/README.md

```diff
@@ -0,0 +1,3 @@
+Before proceeding with running the program, please first install the following packages(for Ubuntu):
```

Comment:
Please follow the template we have for [nginx](https://github.com/unikraft/static-pie-apps/blob/master/nginx/README.md) for the `README` file.